src/components/common/button.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { css } from "@emotion/core";
+import tw from "twin.macro";
+import PropTypes from "prop-types";
+
+const darkStyles = css`
+  ${tw`h-12 w-88 bg-midnight text-white font-bold`}
+`;
+
+const lightGreyStyles = css`
+  ${tw`h-12 w-40 text-midnight border border-midnight font-bold`}
+`;
+
+const lightBlueStyles = css`
+  ${tw`h-12 w-40 text-cobalt border border-cobalt font-bold`}
+`;
+
+const Button = ({ variant, label, onClick }) => (
+  <button css={styleForVariant(variant)} onClick={onClick}>
+    {label}
+  </button>
+);
+
+const styleForVariant = variant =>
+  ({
+    dark: darkStyles,
+    "light-grey": lightGreyStyles,
+    "light-blue": lightBlueStyles,
+  }[variant] ||
+  (() => {
+    throw new Error(`Unknown button variant: ${variant}`);
+  }));
+
+Button.propTypes = {
+  variant: PropTypes.string.isRequired,
```

Comment:
What does this give us which typescript wouldn't?

Reply:
Not sure what you mean? I don't mind switching to typescript in general, just didn't think of doing it as part of this ticket.

Reply:
I just notice that we already have typescript support in the site, which I find a bit easier to read than PropTypes, so I was wondering if PropTypes has a particular advantage over using typescript.

Reply:
Ah I see, none that I know of. We can just agree to write all new code in typescript I guess. And refactor the existing files at some point for consistency